examples/browse/browse.go

```diff
@@ -6,49 +6,164 @@ package main
 
 import (
 	"context"
+	"encoding/csv"
 	"flag"
-	"fmt"
 	"log"
+	"os"
+	"strconv"
 
 	"github.com/gopcua/opcua"
 	"github.com/gopcua/opcua/debug"
 	"github.com/gopcua/opcua/id"
 	"github.com/gopcua/opcua/ua"
+	"github.com/pkg/errors"
```

Comment:
didn't want to use the 1.13 errors?

Reply:
nm. commented on old commit ;)

Reply:
What would this look like with  go1.13 errors? You still don't get stack traces, do you? But thanks for noticing. I've dropped that for now.